### PR TITLE
add headless mode selenium

### PR DESCRIPTION
**Pull request by jpetzke**
### Background
It was annoying that the selenium webdriver always popped up when AutoGPT was performing a browse_website command.

### Changes
I added the headless argument to the selenium options in order to hide the window.

### Test Plan
It was tested multiple times.

### PR Quality Checklist
- [x] My pull request is atomic and focuses on a single change.
- [x] I have thoroughly tested my changes with multiple different prompts.
- [x] I have considered potential risks and mitigations for my changes.
- [x] I have documented my changes clearly and comprehensively.
- [x] I have not snuck in any "extra" small tweaks changes

No further tests needed because the change is too small and does not change functionality.

### CODE REVIEW

**Comment by richbeales** [comment]
This is #1473 implemented in a slightly different manner

File: autogpt/web.py [hunk 1]
```diff
@@ -34,6 +34,7 @@ def scrape_text_with_selenium(url):
     options.add_argument(
         "user-agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.5615.49 Safari/537.36"
     )
+    options.add_argument("--headless")
```
Comment (mezzovide) [inline autogpt/web.py]: better use headless=new
https://developer.chrome.com/articles/new-headless/#new-headless-in-selenium-webdriver

Comment (jpetzke) [reply autogpt/web.py]: what is the difference?

Comment (hdkiller) [reply autogpt/web.py:37]: This article explains the difference: https://developer.chrome.com/articles/new-headless/

Comment (mezzovide) [reply autogpt/web.py]:
```
google-chrome --version
Google Chrome 112.0.5615.121
```

with --headless only :
```
google-chrome --headless
[0415/192703.534998:WARNING:bluez_dbus_manager.cc(247)] Floss manager not present, cannot set Floss enable/disable.
[0415/192703.566803:WARNING:sandbox_linux.cc(393)] InitializeSandbox() called with multiple threads in process gpu-process.
```
with --headless=new no error thrown

**Comment by nponeccop** [comment]
@jpetzke There are conflicts now

**Pull request closed without merging**